### ISSUE TITLE
Missing tag rules not triggering

### ISSUE THREAD

**Issue by mojo-redox** [terraform-linters/tflint#644]
I'm attempting to use the new [tag config rules](https://github.com/terraform-linters/tflint/pull/617) and I'm having trouble getting it to alert me to missing `tags`. 

For example, I have a dynamodb table declared which looks something like:
```terraform
resource "aws_dynamodb_table" "upgrade_locks" {
  name     = "StagingUpgradeLocks"
  hash_key = "StackId"

  write_capacity = 5
  read_capacity  = 5

  attribute {
    name = "StackId"
    type = "S"
  }
}
```

I'd expect this to alert me that the `tags` have not been set, but from the debug output I'm only seeing:
```
17:34:22 runner.go:506: [DEBUG] Walk `aws_dynamodb_table.upgrade_locks.hash_key` attribute
17:34:22 runner.go:506: [DEBUG] Walk `aws_dynamodb_table.upgrade_locks.name` attribute
```

And I receive no warning that the resource is missing `tags`.

I'm using a basic rule in my tflint.hcl config file, which doesn't have anything that I can see would cause an issue:
```hcl
rule "aws_resource_missing_tags" {
  enabled = true
  tags = ["tags_version"]
}
```

**Comment by wata727** [comment]
@mojo-redox Thank you for opening the issue. This is a bug.
I have already fixed this bug and released v0.15.1.

**Comment by nitrocode** [comment]
I'm using the latest tflint v0.15.1 and I see the same issue.

```
$ tflint --version
TFLint version 0.15.1
$ cat main.tf
resource "aws_dynamodb_table" "upgrade_locks" {
  name     = "StagingUpgradeLocks"
  hash_key = "StackId"

  write_capacity = 5
  read_capacity  = 5

  attribute {
    name = "StackId"
    type = "S"
  }
}
$ tflint --aws-region=us-west-2 --deep --module
```

no output
